src/tools/build-manifest/src/main.rs

```diff
@@ -470,6 +470,7 @@ impl Builder {
                 | PkgType::Rls
                 | PkgType::RustAnalyzer
                 | PkgType::Rustfmt
+                | PkgType::Enzyme
```

Comment:
With the enzyme component gone, all build-manifest changes can be reverted too.

Reply:
Well that's also a way to resolve the question I had in a code comment there, great.